example/screens/POIExample.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useContext, useState, useRef } from 'react';
+import { StyleSheet, View, TextInput } from 'react-native';
+
+import * as Maps from 'expo-maps';
+import SwitchContainer from '../components/SwitchContainer';
+import ProviderContext from '../context/ProviderContext';
+
+export default function POIExample() {
+  const provider = useContext(ProviderContext);
+  const ref = useRef<Maps.ExpoMap>(null);
+  const [enablePOISearching, setEnablePOISearching] = useState<boolean>(false);
+  const [enablePOIDisplay, setEnablePOIDisplay] = useState<boolean>(false);
+  const [enablePOIFilter, setEnablePOIFilter] = useState<boolean>(false);
+  const [poiType, setPoiType] = useState<[] | [Maps.POICategoryType]>([]);
+  const [enableQueryCompletions, setEnableQueryCompletions] =
+    useState<boolean>(false);
+  const [text, onChangeText] = useState<string>('');
+  const [placeToSearch, setPlaceToSearch] = useState<string>('');
+  const [enablePlaceSearch, setEnablePlaceSearch] = useState<boolean>(false);
+
+  const appleMapsSearchRequest = 'Centrum Pompidou; Roue 1234';
+  const googleMapsSearchRequest =
+    'Centrum Pompidou; placeId:ChIJoyC4CRxu5kcRRTPcWX5srLc';
+
+  return (
+    <View style={styles.mapContainer}>
+      <Maps.ExpoMap
+        style={{ flex: 1, width: '100%' }}
+        provider={provider}
+        enablePOISearching={enablePOISearching}
+        enablePOIDisplay={enablePOIDisplay}
```

Comment:
I believe you are right. Let's make this feature work the same as the one you implemented for Google Maps. Instead of putting markers on the map, let's display build in points of interests markings. I believe my fix will also improve filtering - I might have misunderstood its use firstly :)

Reply:
@szdziedzic please take a look whether the current version of "EnablePOIs" is something you had in mind :)

Reply:
It's very nice improvement in my opinion, thanks!